beginner_source/deep_learning_60min_blitz.rst

```diff
@@ -8,13 +8,18 @@ Deep Learning with PyTorch: A 60 Minute Blitz
      <iframe width="560" height="315" src="https://www.youtube.com/embed/u7x8RXwLKcA" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
    </div>
 
-Goal of this tutorial:
+What is PyTorch?
+~~~~~~~~~~~~~~~~~~~~~
+PyTorch is a Python-based scientific computing package for two broad purposes:
+
+-  A replacement for NumPy to use the power of GPUs.
+-  A deep learning research platform that provides maximum flexibility and speed
```

Comment:
An automatic differentiation library that is often useful to implement neural networks